docs/alerts.md

```diff
@@ -1 +1,109 @@
-# Alerts
\ No newline at end of file
+
+# 🚨 Alerts 🚨
+
+## Getting Started
+
+This guide provides step-by-step instructions on how to set up alerts and contact points Siglens.
+
+### 📌 Accessing Alerts and Contact Points
+
+
+https://github.com/Davleen14/siglens-docs/assets/114626340/550bc375-2437-4447-9ebb-4393b2bedc7f
+
+
+1. **Navigation:**
+   - Click on the `alerts` button in the side navigation bar. 
+
+   ![alert](/static/img/initial-alert-page.png)
+   ![cp](/static/img/initial-cp-page.png)
+   
+
+   - To add a new alert rule, select the `add new alert rule` button. 
+
+
+### 📊 Creating Alert Rules
+
+1. **Define Alert Rule and Query:**
+   - Enter the details for your alert rule and the corresponding query.
+   ![alert-1](/static/img/alert-1.png)
+
+2. **Set Conditions and Contact Points:**
+   - Specify the conditions for triggering the alert.
+   ![alert-2](/static/img/alert-2.png)
+
+
+   - Select an existing contact point for notifications or create a new one if necessary.
+   ![alert-3](/static/img/alert-3.png)
+   ![alert-3](/static/img/alert-3.1.png)
+
+
+3. **Custom Labels (Optional):**
+   - Add a custom label to your notification for easy identification.
+   ![alert-4](/static/img/alert-4.png)
+
+4. **Save and Verify:**
+   - Save your alert rule. The alert status should now be set to 'firing'.
+   ![alert-5](/static/img/alert-5.png)
+
+
+
+5. **Notification Confirmation:**
+   - For Webhook notifications, confirm message received on the webhook site.
+   ![alert-7](/static/img/alert-7.png)
+
+   - For sending Slack notifications, we follow the same procedure just the contact point changes to slack. 
+   ![alert-6](/static/img/alert-6.png)
+
+   - Check the designated Slack channel for notification.
+   ![alert-8](/static/img/alert-8.png)
+
+### ✏️ Editing and Deleting
+
+1. **Editing Alert Rules:**
+   - To edit an alert rule name, simply select and modify as needed.
+   ![alert-9](/static/img/alert-9.png)
+   ![alert-10](/static/img/alert-10.png)
+   ![alert-11](/static/img/alert-11.png)
+
+2. **Deleting Alert Rules:**
+   - Choose the rule you wish to remove and delete it.
+   ![alert-12](/static/img/alert-12.png)
+   ![alert-13](/static/img/alert-13.png)
+
+
+3. **Managing Contact Points:**
+   - To delete a contact point, ensure it is not linked to any active alert rules. Once free, it can be safely removed.
```

Comment:
`Once free, it can be safely removed.` => `You can not delete contact point associated with any of the alert rules`